cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/AbstractStreamFileAdmin.java

```diff
@@ -273,9 +279,28 @@ public void create(String name, @Nullable Properties props) throws Exception {
     StreamConfig config = new StreamConfig(name, partitionDuration, indexInterval, ttl, streamLocation, null);
     saveConfig(config);
 
+    // Create the notification feeds linked to that stream
+    createStreamFeeds(name);
```

Comment:
Shouldn't be in constructor. Ideally in constructor there shouldn't be any action that involve resources (because it's possible that the service it relies on hasn't been started yet).